release/e2e.py

```diff
@@ -1556,6 +1556,8 @@ def run_test_config(
     app_config["env_vars"]["MATCH_AUTOSCALER_AND_RAY_IMAGES"] = "1"
     app_config["env_vars"]["RAY_bootstrap_with_gcs"] = "1"
     app_config["env_vars"]["RAY_gcs_storage"] = "memory"
+    app_config["env_vars"]["RAY_USAGE_STATS_ENABLED"] = "1"
+    app_config["env_vars"]["RAY_USAGE_STATS_SOURCE"] = "nightly-tests"
```

Comment:
Should we change new infra as well?

Reply:
Hmm doesn't seem to have this logic now. I will just ping Kai once he comes back